lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -6803,6 +6803,37 @@ class DecomposeAtenReshapeAsOp : public OpRewritePattern<AtenReshapeAsOp> {
 };
 } // namespace
 
+namespace {
+// Decompose AtenLinalgNormOp to AtenLinalgVectorNormOp only
+class DecomposeAtenLinalgNormOp : public OpRewritePattern<AtenLinalgNormOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenLinalgNormOp op,
+                                PatternRewriter &rewriter) const override {
+    Location loc = op.getLoc();
+
+    if (op.getDim().getDefiningOp()) {
+      auto dimOp =
+          dyn_cast<Torch::PrimListConstructOp>(op.getDim().getDefiningOp());
```

Comment:
nit: we have a helper for getting a list construct:

https://github.com/llvm/torch-mlir/blob/e85a2a87c5662c26e047a2d93d3ff6216cabdcec/lib/Dialect/Torch/Utils/Utils.cpp#L38